core/feconf.py

```diff
@@ -541,7 +544,7 @@ def get_empty_ratings() -> Dict[str, int]:
 # Port when switching to prod server. Keep this in sync with redis.conf in the
 # root folder. Specifically, REDISPORT should always be the same as the port in
 # redis.conf.
-REDISHOST = 'localhost'
+REDISHOST = 'redis' if OPPIA_IS_DOCKERIZED else 'localhost'
```

Comment:
then we need to have such env variables for every service. I prefer having a single env OPPIA_IS_DOCKERIZED that is only passed to GAE service, instead of having env variable for each service. this will also help in local tests to function as we need to check whether we are in container env or not (for the transition period)

Reply:
done